tests/eval/os.joke

```diff
@@ -0,0 +1,9 @@
+(ns joker.test-joker.os
+  (:require [joker.os :as os]
+            [joker.test :refer [deftest is testing]]))
+
+(deftest test-exec
```

Comment:
It's not consistent in the code base, but I prefer not to use `test-` prefix. `deftest` is clear enough indication that it's a test :). Also, I try to keep tests flat with no nested `testing`. When possible, each `testing` section should be a separate test, makes it easier to run individual tests. So this test could be named `exec-pipe` and `testing` could be removed from it.